frame/balances/src/lib.rs

```diff
@@ -946,39 +947,98 @@ impl<T: Config<I>, I: Instance> Currency<T::AccountId> for Module<T, I> where
 	) -> DispatchResult {
 		if value.is_zero() || transactor == dest { return Ok(()) }
 
-		Self::try_mutate_account(dest, |to_account, _| -> DispatchResult {
-			Self::try_mutate_account(transactor, |from_account, _| -> DispatchResult {
-				from_account.free = from_account.free.checked_sub(&value)
+		let mut maybe_from_acc_call_post_mutation = false;
+		let mut maybe_to_acc_call_post_mutation = false;
+		let mut from_acc: AccountData<T::Balance> = Default::default();
+		let mut to_acc: AccountData<T::Balance> = Default::default();
+
+		T::AccountStore::try_mutate_exists(dest, |maybe_to_acc| -> DispatchResult {
+			let to_acc_is_new = maybe_to_acc.is_none();
+			to_acc = maybe_to_acc.take().unwrap_or_default();
+
+			T::AccountStore::try_mutate_exists(transactor, |maybe_from_acc| -> DispatchResult {
+				let from_acc_is_new = maybe_from_acc.is_none();
+				from_acc = maybe_from_acc
+					.take()
+					.unwrap_or_default();
+
+				from_acc.free = from_acc.free.checked_sub(&value)
 					.ok_or(Error::<T, I>::InsufficientBalance)?;
 
-				// NOTE: total stake being stored in the same type means that this could never overflow
+				// NOTE: total stake being stored in the same type means
+				// that this could never overflow
 				// but better to be safe than sorry.
-				to_account.free = to_account.free.checked_add(&value).ok_or(Error::<T, I>::Overflow)?;
+				to_acc.free = to_acc
+					.free.checked_add(&value)
+					.ok_or(Error::<T, I>::Overflow)?;
 
 				let ed = T::ExistentialDeposit::get();
-				ensure!(to_account.total() >= ed, Error::<T, I>::ExistentialDeposit);
+				ensure!(
+					to_acc.total() >= ed,
+					Error::<T, I>::ExistentialDeposit,
+				);
```

Comment:
I am not sure what is your editor's configured max-width but it is surely too low :D 

For substrate, we use 100, and the CI will rejct more than 120. Please use 100.